tests/acceptance/features/webUIFiles/createFolders.feature

```diff
@@ -19,7 +19,7 @@ Feature: create folders
 
   Scenario: Create a folder with existing name
     When the user creates a folder with the invalid name "simple-folder" using the webUI
-    Then the error message 'Creating folder failed…' should be displayed on the webUI
+    Then the error message 'Creating folder failed… The resource you tried to create already exists' should be displayed on the webUI
```

Comment:
Please consider testing for the title only - the detailed message is server generated and might differ over time, version, environement.

For us it is enough to test if the error message is displayed at this point.